neurom/fst/__init__.py

```diff
@@ -1,177 +1,7 @@
-    'total_area_per_neurite': _nrt.total_area_per_neurite,
-}
-
-NEURONFEATURES = {
-    'soma_radii': _nrn.soma_radii,
-    'soma_surface_areas': _nrn.soma_surface_areas,
-    'soma_volumes': _nrn.soma_volumes,
-    'trunk_origin_radii': _nrn.trunk_origin_radii,
-    'trunk_origin_azimuths': _nrn.trunk_origin_azimuths,
-    'trunk_origin_elevations': _nrn.trunk_origin_elevations,
-    'trunk_section_lengths': _nrn.trunk_section_lengths,
-    'trunk_angles': _nrn.trunk_angles,
-    'trunk_vectors': _nrn.trunk_vectors,
-    'sholl_frequency': _nrn.sholl_frequency,
-}
-
-
-def register_neurite_feature(name, func):
-    '''Register a feature to be applied to neurites
-
-    Parameters:
-        name: name of the feature, used for access via get() function.
-        func: single parameter function of a neurite.
-    '''
-    if name in NEURITEFEATURES:
-        raise NeuroMError('Attempt to hide registered feature %s' % name)
-
-    def _fun(neurites, neurite_type=_ntype.all):
-        '''Wrap neurite function from outer scope and map into list'''
-        return list(func(n) for n in _ineurites(neurites, filt=_is_type(neurite_type)))
-
-    NEURONFEATURES[name] = _fun
-
-
-def get(feature, obj, **kwargs):
-    '''Obtain a feature from a set of morphology objects
-
-    Parameters:
-        feature(string): feature to extract
-        obj: a neuron, population or neurite tree
-        **kwargs: parameters to forward to underlying worker functions
-
-    Returns:
-        features as a 1D or 2D numpy array.
-
-    '''
-
-    feature = (NEURITEFEATURES[feature] if feature in NEURITEFEATURES
-               else NEURONFEATURES[feature])
-
-    return _np.array(list(feature(obj, **kwargs)))
-
-
-_INDENT = ' ' * 4
-
-
-def _indent(string, count):
-    '''indent `string` by `count` * INDENT'''
-    indent = _INDENT * count
-    ret = indent + string.replace('\n', '\n' + indent)
-    return ret.rstrip()
-
-
-def _get_doc():
-    '''Get a description of all the known available features'''
-    def get_docstring(func):
-        '''extract doctstring, if possible'''
-        docstring = ':\n'
-        if func.__doc__:
-            docstring += _indent(func.__doc__, 2)
-        return docstring
-
-    ret = ['\nNeurite features (neurite, neuron, neuron population):']
-    ret.extend(_INDENT + '- ' + feature + get_docstring(func)
-               for feature, func in sorted(NEURITEFEATURES.items()))
-
-    ret.append('\nNeuron features (neuron, neuron population):')
-    ret.extend(_INDENT + '- ' + feature + get_docstring(func)
-               for feature, func in sorted(NEURONFEATURES.items()))
-
-    return '\n'.join(ret)
-
-
-get.__doc__ += _indent('\nFeatures:\n', 1) + _indent(_get_doc(), 2)  # pylint: disable=no-member
+from neurom.features import NEURITEFEATURES, NEURONFEATURES, get, register_neurite_feature
+print('neurom.fst is being deprecated, please consider replacing it by neurom.features')
```

Comment:
A couple of suggestions here and for other similar locations:

- Should this be a warning?
- If possible, note when it will be removed: next major version bump?
- Remove 'please consider' and just say what they need to change